azurerm/internal/services/frontdoor/frontdoor_firewall_policy_resource.go

```diff
@@ -167,6 +167,7 @@ func resourceArmFrontDoorFirewallPolicy() *schema.Resource {
 											string(frontdoor.RequestHeader),
 											string(frontdoor.RequestMethod),
 											string(frontdoor.RequestURI),
+											string(frontdoor.SocketAddr),
```

Comment:
can we [add this to the documentation too](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/r/frontdoor_firewall_policy.html.markdown#L170)?